storer/deno_kv/deno_kv_storer.ts

```diff
@@ -0,0 +1,289 @@
+import type { Award, MintedPerk } from "../../perks/mod.ts";
+import type {
+  AwardQuery,
+  Diagnosis,
+  ListQuery,
+  MintQuery,
+  PerkQuery,
+  PreuseQuery,
+  RevokeQuery,
+  StoredSummary,
+  Storer,
+  UnmintQuery,
+} from "../mod.ts";
+
+/**
+ * DenoKVStorer uses Deno KV to implement the Storer interface.
+ *
+ * See:
+ * https://deno.land/api?unstable=&s=Deno.Kv
+ */
+export class DenoKVStorer implements Storer {
```

Comment:
`DenoKVStorer` inaccurately implements the `Deno.Kv` API. Refer to <https://deno.com/blog/kv#atomic-transactions>.

Reply:
Resolved in <https://github.com/acmcsufoss/perks/pull/1/commits/c70c1e608bf1d6f4dd8f6f33020ccef3b25a7981>.